todoApp.js

```diff
@@ -0,0 +1,65 @@
+var express = require('express')
```

Comment:
Read `Resource Oriented Design` on how to create and return a resource response.